src/components/__tests__/Button-test.js

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import Button from '../Button';
+import TestRenderer from 'react-test-renderer';
+
+const props = {
+  pin: 6,
+  onChange: jest.fn(),
+  onDown: jest.fn(),
+  onUp: jest.fn(),
+  internalPullup: false,
+};
+
+let component;
+let view;
```

Comment:
nit: I’m personally not a fan of beforeEach or global mutating variables in tests. This isn’t saving much by doing this over binding this variables in each test, and doing so inside each `it` block removes any concern of race conditions, external effects, etc.

Reply:
I'll take care of that soon - can't guarantee this week due to personal commitments but soon.

Reply:
it’ll be a race. If I decide to work on this some night I’ll land this as is and make the adjustment in a quick followup.

Reply:
Let's give those changes a shot - let me know your thoughts on them!